### ISSUE TITLE
Why not embed  '<script src="various.js"></script>'  into templateUrl "html" document ?

### ISSUE THREAD

**Issue by marcusbo** [angular-ui/ui-router#53]
Hi,

Something we do in CRUD environments is to include JavaScript routines for a specific `<div>` partial, so that we can control everything going on inside the partial, for example:

-------  start html document ------

```
<script type="text/javascript">
    function MyCtrl($scope) { 
       . . . JavaScript functions . . .
    }
</script>
<div ng-controller="MyCtrl">
      . . . various . . .
</div>
```

-------  end html document -------

Notice that MyCtrl is a controller that is inside the HTML document.

This enables us to have different developers writing specific business logic for each CRUD section.

I would like to use the  "templateUrl"  option so that we can load an `xxx.html`, in which we can add as many `<script type='text/javascript'>` with embedded controllers and other JavaScript code bits.

In other words, the classical ui-router just needs:

``` $stateProvider
  .state('report',{
    views: {
      'tabledata': {
        templateUrl: 'MyXXX.html',
       NO NEED FOR CONTROLLER HERE, 
      BECAUSE JS CODE IS EMBEDDED IN 'MyXXX.html'
     }
      },
      'graph': {
        templateUrl: 'report-graph.html',
        controller: function($scope){... controller stuff... }
      },
    }
  })
```

Thanks,

Marcus
